### ISSUE TITLE
Is QuickBox dead?

### ISSUE THREAD

**Issue by gexala** [QuickBox/QB#57]
Is quickbox dead? The plaza is down, and webpage is down?

**Comment by taterzero** [comment]
Was wondering the same thing. Hopefully someone will let us all now.

**Comment by seedboxer** [comment]
Would also like to know, has this awesome project been abandoned due to lack of funding?

**Comment by tylerhoot** [comment]
everyone the rebuilt by Liara is at [](https://swizzin.ltd/)  [](https://github.com/liaralabs/swizzin)

**Comment by k0nsl** [comment]
It would have been a good idea to officially announce the death of the project publicly somewhere (e.g in 'README.md').

**Comment by seedboxer** [comment]
People not sure if it is actually dead, this is the thing.

**Comment by taterzero** [comment]
so what happens to all of us that paid for the use of the DNS? does that mean it won't work anymore? So does this mean quickbox won't work after awhile? Some pretty legit questions.

**Comment by tylerhoot** [comment]
Yes I agree with that and not leaving​ us hanging or let us know what is
going on I have 2x boxes to migrate

**Comment by tylerhoot** [comment]
Those people might be out of luck. But this is all unknown intill the
creator can confirm what is going on

**Comment by seedboxer** [comment]
@taterzero 

Is it working now ? If so i suppose there is some hope the main site may return.

**Comment by taterzero** [comment]
yes stuff is working now but my question is for how long?

**Comment by seedboxer** [comment]
@taterzero 

We all hope forever and we hope it will return.

**Comment by dtech4you** [comment]
AS of right now the DNS will stay active as far as i am aware. Also yes it is sad that QB is shutdown for now due to funding being little to none. but later on we might come back with the site and the guru installs. if anyone really needs help with installs I will be around here and in the chat.quickbox.io for people who need help it's still active as a member is hosting it on a vps for us. and if that stops i can move it to a local server to run. that being said i am really busy as i am doing a new job. so my time will cost money.

**Comment by gexala** [comment]
this is just sad! but understandable.  What´s your opinion about swizzin? Where is the rest of the qb crew? Only you left?

Alternative to quickbox;
https://bytesized-hosting.com/connect (will probably cost some money soon) 
https://zackreed.me/docker-how-and-why-i-use-it/ - Good docker guide

**Comment by dtech4you** [comment]
bates is around the chat sometimes not often though but yeah pretty much just me right now... and swizzin looks cool it's just a addon to quickbox

**Comment by taterzero** [comment]
question the quickbox script is still good right? I mean no worry of the box itself just stop working is there?

**Comment by seedboxer** [comment]
@dtech4you 

Hi i did dontate to this as it was so useful/good.

Could you consider putting the project behind a paywall. £2-3, $2-3, or £2-3 Euro per month, or payable per year etc.

It has all the features i need, its only updates  and maintaining.  Would this be viable ?

**Comment by dtech4you** [comment]
updates till can be posted people are more than welcome to push updates and i will add them using github as it always has been i just wont have time to do any coding and i know JMsolo wont for awhile.

**Comment by reinierkors** [comment]
Why don't you host it on github? No need for expensive servers, get rid of the community forum.

**Comment by taterzero** [comment]
Is there a way to have the servers using the quickbox dns removed?

**Comment by Neonlinx** [comment]
This is very sad. QuickBox was (is) the best project of its kind!!! I hope everything returns to normal as soon as possible. I would also accept gladly to see some commercial ads in the Plaza, if that can help...
Anyway....I can't access IRC channel (timedout)...it's only me or we lost also the chat? where we can remain in contact??

**Comment by RajPatel0929** [comment]
I think instated of closing this project, should have kept open source, and it's free... this project was best now so sad to go :(

but thanks for who ever worked on it.. you guys are the best :D

**Comment by LWSRep** [comment]
Quixkbox isn't gone it is back already.
Visit their website and read the announcement

**Comment by JMSDOnline** [comment]
It's true fam, we're alive and well. We experienced some difficulties that resulted in a rather sudden downtime. Namely, the server we had the forums on decided to check out (permanently). It was compounded by the fact that the funding needed simply wasn't there. Time is still a slight issue as we have expressed. I have several projects burning (that pay my bills and feed my family) and this does consume a great deal of time... however, this doesn't mean that I haven't been working on some changes and feature additions... locally. 

We're excited to be back and we'll continue to give when and where we can.

Thanks again to everyone for sharing the love and encouraging us to keep the dream alive. We will try to not let you guys down.
